test/PooltogetherGovernor.t.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {PooltogetherGovernorTest} from "test/helpers/PooltogetherGovernorTest.sol";
+
+contract Constructor is PooltogetherGovernorTest {
+  function testFuzz_CorrectlySetsAllConstructorArgs(uint256 _blockNumber) public {
+    assertEq(governorBravo.name(), "Pooltogether Governor Bravo");
+    assertEq(address(governorBravo.token()), POOL_TOKEN);
+    // These values were all copied directly from the mainnet alpha governor at:
+    // 0xB3a87172F555ae2a2AB79Be60B336D2F7D0187f0
+    assertEq(INITIAL_VOTING_DELAY, 1);
+    assertEq(INITIAL_VOTING_PERIOD, 28_800);
+    assertEq(INITIAL_PROPOSAL_THRESHOLD, 1000e18);
+    // forgefmt: disable-end
+    assertEq(governorBravo.votingDelay(), INITIAL_VOTING_DELAY);
+    assertEq(governorBravo.votingPeriod(), INITIAL_VOTING_PERIOD);
```

Comment:
Let's add an assertion that any delays are set below some upper bound threshold of reasonable time (maybe 6 weeks or something like that). One way an error could be made is if someone were to miscalculate while updating these values and set them to something impossibly long. Testing would all work but in real life the Governor could be bricked for 4 billion years or something. So something like `assertLt(governorBravo.votingDelay(), MAXIMUM_REASONABLE_TIME_PERIOD)

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/9/commits/2f07ad167101bbab14f62020236c56e22f6c1b3c

Reply:
Hmm, I see the constant but I don't see the assertions. Am I missing something @alexkeating?

Reply:
You see correct, I am not sure what happened, but will get the changes up

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/9/commits/4a590b49a0a8ba2137e64890da3fe0745c379d57